recipe/meta.yaml

```diff
@@ -25,14 +25,10 @@ requirements:
 
 test:
   commands:
-    - set vex
-    - which nim
+    - command -v nim
     - nim -v
-    - readlink -f $(which nim)
-    - ls -larth
-    - ls -l $(dirname $(readlink -f $(which nim)))
-#    - echo 'echo "hello"' >| foo.nim
-#    - nim c -r foo.nim
+    - echo 'echo "hello"' >| foo.nim
```

Comment:
Please put back `ls -larth`. That helps me understand what's going on.

Reply:
This is ready for merging.
Apart from this remark of yours. Which path do you actually want to `ls -larth`? As it was, you'd just list the contents of the temporary test directory, but nothing from the prefix itself.